Docs/sphinx/manual/Model.rst

```diff
@@ -118,6 +118,12 @@ For the standard ideal gas EOS, the divergence constraint on velocity becomes:
     \nabla \cdot \boldsymbol{u} &= \frac{1}{\rho c_p T} \left(\nabla \cdot \lambda\nabla T - \sum_m \boldsymbol{\Gamma_m} \cdot \nabla h_m \right) \\
     &- \frac{1}{\rho} \sum_m \frac{W}{W_m} \nabla \cdot \boldsymbol{\Gamma_m} + \frac{1}{\rho}\sum_m \left(\frac{W}{W_m} - \frac{h_m}{c_p T} \right) \dot \omega \equiv S .
 
+In addition to the flow equations, `PeleLMeX` can also solve for a set of quantities that are neither advected nor diffused, satisfying:
+
+.. math::
+
+    \frac{\partial B_k}{\partial t} = S_{\text{ext},B_k}.
```

Comment:
Should be `d` rather than `\partial` because these only change with time?

Reply:
$B_k = B_k(x_j,t)$ so the \partial is needed.